tests/CMakeLists.txt

```
@@ -0,0 +1,186 @@
+add_fdb_test(TEST_FILES fast/AtomicBackupToDBCorrectness.txt)
+add_fdb_test(TEST_FILES fast/AtomicOps.txt)
+add_fdb_test(TEST_FILES fast/AtomicOpsApiCorrectness.txt)
+add_fdb_test(TEST_FILES fast/BackupCorrectness.txt)
+add_fdb_test(TEST_FILES fast/BackupCorrectnessClean.txt)
+add_fdb_test(TEST_FILES fast/BackupToDBCorrectness.txt)
+add_fdb_test(TEST_FILES fast/BackupToDBCorrectnessClean.txt)
+add_fdb_test(TEST_FILES fast/CloggedSideband.txt)
+add_fdb_test(TEST_FILES fast/ConstrainedRandomSelector.txt)
+add_fdb_test(TEST_FILES fast/CycleAndLock.txt)
+add_fdb_test(TEST_FILES fast/CycleTest.txt)
+add_fdb_test(TEST_FILES fast/FuzzApiCorrectness.txt)
+add_fdb_test(TEST_FILES fast/FuzzApiCorrectnessClean.txt)
+add_fdb_test(TEST_FILES fast/IncrementTest.txt)
+add_fdb_test(TEST_FILES fast/InventoryTestAlmostReadOnly.txt)
+add_fdb_test(TEST_FILES fast/InventoryTestSomeWrites.txt)
+add_fdb_test(TEST_FILES fast/LongStackWriteDuringRead.txt)
+add_fdb_test(TEST_FILES fast/LowLatency.txt)
+add_fdb_test(TEST_FILES fast/MemoryLifetime.txt)
+add_fdb_test(TEST_FILES fast/MoveKeysCycle.txt)
+add_fdb_test(TEST_FILES fast/RandomSelector.txt)
+add_fdb_test(TEST_FILES fast/RandomUnitTests.txt)
+add_fdb_test(TEST_FILES fast/SelectorCorrectness.txt)
+add_fdb_test(TEST_FILES fast/Sideband.txt)
+add_fdb_test(TEST_FILES fast/SidebandWithStatus.txt)
+add_fdb_test(TEST_FILES fast/SwizzledRollbackSideband.txt)
+add_fdb_test(TEST_FILES fast/SystemRebootTestCycle.txt)
+add_fdb_test(TEST_FILES fast/TaskBucketCorrectness.txt)
+add_fdb_test(TEST_FILES fast/TimeKeeperCorrectness.txt)
+add_fdb_test(TEST_FILES fast/Unreadable.txt)
+add_fdb_test(TEST_FILES fast/VersionStamp.txt)
+add_fdb_test(TEST_FILES fast/Watches.txt)
+add_fdb_test(TEST_FILES fast/WriteDuringRead.txt)
+add_fdb_test(TEST_FILES fast/WriteDuringReadClean.txt)
+add_fdb_test(TEST_FILES rare/CheckRelocation.txt)
+add_fdb_test(TEST_FILES rare/ClogUnclog.txt)
+add_fdb_test(TEST_FILES rare/CloggedCycleWithKills.txt)
+add_fdb_test(TEST_FILES rare/ConflictRangeCheck.txt)
+add_fdb_test(TEST_FILES rare/ConflictRangeRYOWCheck.txt)
+add_fdb_test(TEST_FILES rare/CycleRollbackClogged.txt)
+add_fdb_test(TEST_FILES rare/CycleWithKills.txt)
+add_fdb_test(TEST_FILES rare/FuzzTest.txt)
+add_fdb_test(TEST_FILES rare/InventoryTestHeavyWrites.txt)
+add_fdb_test(TEST_FILES rare/LargeApiCorrectness.txt)
+add_fdb_test(TEST_FILES rare/LargeApiCorrectnessStatus.txt)
+add_fdb_test(TEST_FILES rare/RYWDisable.txt)
+add_fdb_test(TEST_FILES rare/RandomReadWriteTest.txt)
+add_fdb_test(TEST_FILES rare/SwizzledLargeApiCorrectness.txt)
+add_fdb_test(
+  TEST_FILES restarting/CycleTestRestart-1.txt
+             restarting/CycleTestRestart-2.txt)
+add_fdb_test(
+  TEST_FILES restarting/StorefrontTestRestart-1.txt
+             restarting/StorefrontTestRestart-2.txt)
+add_fdb_test(
+  TEST_FILES restarting/from_5.1.7/DrUpgradeRestart-1.txt
+             restarting/from_5.1.7/DrUpgradeRestart-2.txt)
```

```suggestion
             restarting/from_5.1.7/DrUpgradeRestart-2.txt IGNORE)
```

The from_* tests are meant to test the specific upgrades, so these would currently repeat existing tests.  I don't think this is worth fixing... It'd be absurdly difficult to manage a bunch of versions of fdbserver in ctest.  Annotating the version requirements by folder name was a hack anyway.